src/selectors/entities/channels.js

```diff
@@ -1156,3 +1156,44 @@ export const getSortedDirectChannelWithUnreadsIds: (GlobalState, Channel, boolea
         return filterChannels(unreadChannelIds, favoritePreferences, directChannelIds, false, favoritesAtTop);
     },
 );
+
+export const getDefaultChannelForTeams: (GlobalState) => RelationOneToOne<Team, Channel> = createSelector(
```

Comment:
Worth adding some function-level documentation describing the "concept" of default channels here?

Reply:
Yes, maybe.